.github/workflows/image.yaml

```diff
@@ -0,0 +1,65 @@
+on:
+  push:
+    branches:
+      - "master"
```

Comment:
how about `release` branch?

Reply:
That's handled by the tags trigger below.